packages/i18n/src/fi.ts

```diff
@@ -0,0 +1,83 @@
+
+// prettier-ignore
+const language: Language = {
+  code:               'fi',
+  schema:             (issue) => `Virheellinen tyyppi: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+  specific: {
+    base64:           (issue) => `Virheellinen Base64: Saatiin ${issue.received}`,
+    bic:              (issue) => `Virheellinen BIC: Saatiin ${issue.received}`,
+    bytes:            (issue) => `Virheelliset tavut: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    check:            (issue) => `Virheellinen syöte: Saatiin ${issue.received}`,
+    checkAsync:       (issue) => `Virheellinen syöte: Saatiin ${issue.received}`,
+    checkItems:       (issue) => `Virheellinen osio: Saatiin ${issue.received}`,
+    checkItemsAsync:  (issue) => `Virheellinen osio: Saatiin ${issue.received}`,
+    creditCard:       (issue) => `Virheellinen luottokortti: Saatiin ${issue.received}`,
+    cuid2:            (issue) => `Virheellinen Cuid2: Saatiin ${issue.received}`,
+    decimal:          (issue) => `Virheellinen desimaaliluku: Saatiin ${issue.received}`,
+    digits:           (issue) => `Virheellinen numeromäärä: Saatiin ${issue.received}`,
+    email:            (issue) => `Virheellinen sähköposti: Saatiin ${issue.received}`,
+    emoji:            (issue) => `Virheellinen emoji: Saatiin ${issue.received}`,
+    empty:            (issue) => `Virheellinen pituus: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    endsWith:         (issue) => `Virheellinen loppu: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    everyItem:        (issue) => `Virheellinen osio: Saatiin ${issue.received}`,
+    excludes:         (issue) => `Virheellinen sisältö: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    finite:           (issue) => `Virheellinen äärellinen luku: Saatiin ${issue.received}`,
+    graphemes:        (issue) => `Virheellinen grafeemimäärä: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    hash:             (issue) => `Virheellinen hash: Saatiin ${issue.received}`,
+    hexColor:         (issue) => `Virheellinen heksaväri: Saatiin ${issue.received}`,
+    hexadecimal:      (issue) => `Virheellinen heksadesimaali: Saatiin ${issue.received}`,
+    imei:             (issue) => `Virheellinen IMEI: Saatiin ${issue.received}`,
+    includes:         (issue) => `Virheellinen sisältö: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    integer:          (issue) => `Virheellinen kokonaisluku: Saatiin ${issue.received}`,
+    ip:               (issue) => `Virheellinen IP: Saatiin ${issue.received}`,
+    ipv4:             (issue) => `Virheellinen IPv4: Saatiin ${issue.received}`,
+    ipv6:             (issue) => `Virheellinen IPv6: Saatiin ${issue.received}`,
+    isoDate:          (issue) => `Virheellinen päivämäärä: Saatiin ${issue.received}`,
+    isoDateTime:      (issue) => `Virheellinen päivämäärä-aika: Saatiin ${issue.received}`,
```

Comment:
"Päivämäärä-aika" is not Finnish. Perhaps "ajanhetki" or "ajankohta".

Reply:
The product of date (päivämäärä) and a time of day (kellonaika) is a datetime (ajanhetki).

Of course there is some ambiguity if it's local or associated with a time-zone. But the same ambiguity is present in the English version.

Anyway, I've never seen the word "päivämäärä-aika" in Finnish, and it feels like clumsy anglicism.

But go do whatever you feel is best. I'm just giving my 5 cents and you're free to ignore that if I seem too pedantic. I don't wish to force anything upon you, just giving feedback from one perspective.

Reply:
> Virheellinen ISO-päivämäärä- ja aika

Isn't the hyphen in the wrong place here? That would imply that we have a prefix of a compound word. Like "koiran- ja kissanruokaa" is short for "koiranruokaa and kissanruokaa".

But perhaps _"Virheellinen ISO-päivämäärä ja -aika"_ would work.